client/stylesheets/_modal.scss

```diff
@@ -212,17 +212,17 @@
 }
 
 /* Radio Buttons */
-.modOfferRequestPaymentWrap ul li {
+.radioWrap ul li {
 	clear: both;
 	color: #999 !important;
 	display: inline-block;
 }
 
-.modOfferRequestPaymentWrap ul li:nth-child(2), .modOfferRequestPaymentWrap ul li:nth-child(3) {
+.radioWrap ul li:nth-child(2), .radioWrap ul li:nth-child(3) {
 	margin-left: 15px;
 }
 
-.modOfferRequestPaymentWrap ul li label {
+.radioWrap ul li label {
```

Comment:
Merge rule `.radioWrap ul li label` with rule on line 215<br>Selector should have depth of applicability no greater than 2, but was 4<br>Selector `radioWrap` should be written in lowercase with hyphens